scripts/superdesk/editor2/editor.js

```diff
@@ -963,7 +963,6 @@ angular.module('superdesk.editor2', [
                         // if config.multiBlockEdition is true, add Embed and Image button to the toolbar
                         if (scope.config.multiBlockEdition) {
                             // this dummy imageDragging stop preventing drag & drop events
```

Comment:
Then this comment can be removed. And does everything works fine with image drag&drop now ?